WMF Framework/ExploreCardCollectionViewCell.swift

```diff
@@ -5,23 +5,35 @@ public protocol CardContent {
     func contentHeight(forWidth: CGFloat) -> CGFloat
 }
 
+fileprivate protocol CardBackgroundViewDelegate: AnyObject {
+    var titleAreaTapped: Bool { get set }
+}
+private class CardBackgroundView: UIView {
+    fileprivate weak var delegate: CardBackgroundViewDelegate?
+    override public func hitTest(_ point: CGPoint, with event: UIEvent?) -> UIView? {
+        delegate?.titleAreaTapped = point.y < 0
```

Comment:
Could the `0` be changed to something like `frame.maxY` so that it covers the areas above the bottom of the card. 

This is a slight change - In testing, I realized that tapping the margin between the card and the edge of the screen* results in going to a novel random article.

*In real life, a finger is unlikely to hit this. But you never know, and if people are using other types of devices... can't hurt?

Reply:
That's a great catch! I hadn't realized the margins weren't part of the content itself.

I have it working, but of course, it's a little more complex since the card needs to report the right offset to the background view when hit testing happens.

I'll look at doing the hit testing override in the cell again. The method will get called more often (even when the content is tapped) but that should not add much overhead, but it cuts down on the complexity communicating to the background view.